browser/components/molecules/PendingPrrrs/index.js

```diff
@@ -28,9 +31,22 @@ export default class PendingPrrrs extends Component {
         console.warn('Claim Error')
         console.error(error)
         this.setState({error})
+        this.scheduleErrorRemoval()
       })
   }
 
+  scheduleErrorRemoval() {
+    if (this.state.errorTimeoutId) {
+      window.clearTimeout(this.state.errorTimeoutId)
```

Comment:
you do not need the `window.` part. `setTimeout` and `clearTimeout` are global variables.